testing_database/creating-the-testing-database.md

```diff
@@ -0,0 +1,34 @@
+# Creating the testing database
+At the writing this document, the testing database is created using Discourse v3.2.0.
+
+## Creating the database
+First of all we need to deploy the Discourse following the [tutorial](https://github.com/canonical/discourse-k8s-operator/blob/main/docs/tutorial.md).
```

Comment:
When this PR lands, deploying discourse by following the regular tutorial would give us v3.3.0 wouldn't it?

Reply:
Changed it to 
`At the writing this document, the testing database is created using Discourse v3.2.0 and used to test migration from v3.2.0 to v3.3.0.`

to make it more clear.

Reply:
Yes, but as soon as this PR lands this documentation will be obsolete, so I don't think it's a good idea to land it as is.

Reply:
Makes sense, the goal of this documentation is to explain how to create a testing db and the steps are not effected by the version. What should I do in this case?